.github/ct-lint.yaml

```diff
@@ -0,0 +1,19 @@
+# In this repository there are two folder with charts that need to be released. One can think that it would be easier to  simply at to this file
+# a chart-dirs clause pointing to both of the folders at the same time:
+
+# ```yaml
+# chart-dirs:
+#   - library
+#   - charts
+# ```
+
+# In that case `ct install` is going to try to install the library chart and is going to fail. That is the reason there are two `ct.yaml` files, one
+# for the step that lint that includes all the folders at the same time and other of the installation step that test installability
+
+chart-dirs:
```

Comment:
Could we have them in the same directory and add with the exclude https://github.com/bitnami/charts/blob/596d26b70b69b438e15047e83710345ee49f8f57/.github/ct-install.yaml#L6

to the ct-install?

Reply:
I said that we could not do that because I did not read the source code properly: https://github.com/helm/chart-testing/blob/main/pkg/config/config.go#L59

On the other hand, I would not change it because if we do it is going to try to install the `CHART-TEMPLATE` which is going to fail. I would also want to lint `common-library` so pipeline tests if we have properly bumped the version and such.

Good catch nevertheless...

Reply:
Oh, and the one that has no exceptions is the chart release: https://github.com/helm/chart-releaser/blob/main/pkg/config/config.go#L40-L61

So it is not that I did not read properly but that in my mind we are talking about the same tool :)

Reply:
I am fine with dividing them if it is not a standard to keep them under charts
If we want to keep the same we could do the following:

One for the install
```
exclude
 - common-library
 - CHART-TEMPLATE
chart-repos:
  - kube-state-metrics=https://kubernetes.github.io/kube-state-metrics
  - pixie-operator=https://pixie-operator-charts.storage.googleapis.com
```

One for the linting
```
chart-repos:
  - kube-state-metrics=https://kubernetes.github.io/kube-state-metrics
  - pixie-operator=https://pixie-operator-charts.storage.googleapis.com
```